functions-and-scope.js

```diff
@@ -21,16 +35,36 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Schrijf een functie genaamd cumLaude, die een array van cijfers verwacht (zoals grades) en het aantal Cum laude studenten teruggeeft. Gebruik hiervoor jouw antwoord van 1a.
 // Zorg ervoor dat jouw functie ook werkt als we een andere array met eindcijfers willen checken, zoals bijvoorbeeld: [6, 4, 5] of [8, 9, 4, 6, 10].
 // Log het antwoord in de terminal.
+function cumLaude(arr) {
+    let counter = 0
+    for (let i = 0; i < arr.length; i++) {
+        if (arr[i] >= 8) {
+            counter = counter + 1
+        }
+    }
+    return counter;
+}
+
+console.log(cumLaude([6, 4, 5]))
 
 // ---- Verwachte uitkomsten:
 // cumLaude(grades) geeft 6
 // cumLaude([6, 4, 5]) geeft 0
 // cumLaude([8, 9, 4, 6, 10]) geeft 3
 
 
-
-
 /* Opdracht  2: Gemiddeld cijfer */
+function avarage(arr) {
+    let total = 0;
+    let totalGrades = 0;
```

Comment:
De uitkomst weet je in principe vooraf, namelijk
```suggestion
    let totalGrades = arr.length;
```